src/App.tsx

```diff
@@ -8,7 +8,105 @@
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getUser } from './api';
```

Comment:
The `api.ts` file is missing, which is essential for fetching data correctly from the API. Please ensure it is included in your submission.